Fix languagetool findings (#299)
SAP/cloud-sdk: docs/js/features/odata/common/entity/from-json.mdx

> In some cases it makes sense for you to store your data as a JSON object, that is based on the entity type, i.e. using the property names of the entity class as properties of your object.

Reviewer: ```suggestion
Sometimes, it makes sense for you to store your data as a JSON object, that is based on the entity type.
For example, when using the property names of the entity class as properties of your object.
```
